### PR TITLE
Added alot and casper can use my new function: modlog

### PR DESCRIPTION
**Pull request by tovade**

### CODE REVIEW

File: src/commands/admin/prefix.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const { setServerPrefix, getServerPrefix } = require("../../utils/functions");
 const { ownerId } = require("../../../config.json");
+const { MessageEmbed } = require("discord.js");
```
Comment (casperiv0) [inline src/commands/admin/prefix.js]: Please remove this line as It's not being used.

Comment (tovade) [reply src/commands/admin/prefix.js]: oh i thought u were gonna use it for modlog

Comment (tovade) [reply src/commands/admin/prefix.js]: well ok then

Comment (tovade) [reply src/commands/admin/prefix.js]: done

**Pull request merged**